app/poros/save_request/skip_checker.rb

```diff
@@ -0,0 +1,21 @@
+class SaveRequest::SkipChecker
+  def initialize(params:)
+    @controller = params['controller'].presence!
+    @action = params['action'].presence!
+    @params = params.presence!
+  end
+
+  def skip?
+    case [@controller, @action, @params.symbolize_keys]
+    in (
+      ['health_checks', _, _] |
+      ['anonymous', _, _] |
+      ['blog', 'assets', _] |
+      [_, _, { uptime_robot: _ }]
```

Comment:
This might be one of the first times that I have leveraged Ruby pattern matching? I think it works pretty well here.